compose-builder/get-api-gateway-token.sh

```diff
@@ -24,16 +24,32 @@ if [ "$DEV" = "-dev" ]; then
   CORE_EDGEX_VERSION=0.0.0
 fi
 
-rm -rf keys
-mkdir keys
-openssl ecparam -name prime256v1 -genkey -noout -out keys/ec256.key 2> /dev/null
-cat keys/ec256.key | openssl ec -out keys/ec256.pub 2> /dev/null
+# Init key dir
+GW_KEY_DIR=/tmp/edgex-gateway-keys
+
+rm -rf ${GW_KEY_DIR}
+mkdir -p ${GW_KEY_DIR}
+
+# Build Gateway Key
+openssl ecparam -name prime256v1 -genkey -noout -out ${GW_KEY_DIR}/gateway.key 2> /dev/null
+openssl ec -in ${GW_KEY_DIR}/gateway.key -pubout -out ${GW_KEY_DIR}/gateway.pub 2> /dev/null
+
+# JWT File
+JWT_FILE=/tmp/edgex/secrets/edgex-security-proxy-setup/kong-admin-jwt
+JWT_VOLUME=/tmp/edgex/secrets/edgex-security-proxy-setup
+#JWT=`cat ${JWT_FILE}`
```

Comment:
remove dead code

Reply:
You are just requesting that I remove the JWT var - correct?

Reply:
this one line, # is the syntax for comment out in shell script
and also remove `ID=`uuidgen` line as it is duplicated to the line `export ID=uuidgen`